app/controllers/shows_controller.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+class ShowsController < ApplicationController
+  before_action except: %i[index show] do
+    redirect_to movies_path unless manager?
+  end
+  before_action :find_show, only: %i[show destroy]
+
+  def index
+    @movies = Movie.all
+  end
+
+  def show; end
```

Comment:
Line 13 need to be fixed

Reply:
this is fine unless the endpoint is unused then it could be deleted

Reply:
This endpoint is used, so I won't delete it. I got that from an Rubocop autocorrection.